### PR TITLE
updated consent

### PR DESCRIPTION
**Pull request by malidgi**

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/mtp/data-privacy.md [hunk 1]
```diff
@@ -28,12 +28,9 @@ search.appverid:
...
+By using the Microsoft Threat Protection you consent to the following terms:
 
+By turning on Microsoft Threat Protection (MTP), you consent to the following terms. Your applicable Customer Data as defined in the Online Service Terms (OST) will be transferred from other Microsoft services into MTP. Use of your Customer Data in MTP will follow the data handling standards and commitments for Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP). You acknowledge that such commitments may differ from the services from which that Customer Data is transferred. Further, Customer Data stored in MTP will be stored at rest in the Geo you selected for storage of your MDATP Customer Data, which may differ from the Geo you selected in connection with other services. Microsoft will not transfer the Customer Data outside of such Geo except as noted in the Data Location section of the [Microsoft Trust Center](https://www.microsoft.com/trust-center)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/data-privacy.md:33]:
```suggestion
By turning on Microsoft Threat Protection (MTP), you consent to the following terms. Your applicable Customer Data as defined in the Online Service Terms (OST) will be transferred from other Microsoft services into MTP. Use of your Customer Data in MTP will follow the data handling standards and commitments for Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP). You acknowledge that such commitments may differ from the services from which that Customer Data is transferred. Further, Customer Data stored in MTP will be stored at rest in the Geo you selected for storage of your MDATP Customer Data, which may differ from the Geo you selected in connection with other services. Microsoft will not transfer the Customer Data outside of such Geo except as noted in the Data Location section of the [Microsoft Trust Center](https://www.microsoft.com/trust-center).
```

Comment (JohanFreelancer9) [reply microsoft-365/security/mtp/data-privacy.md:33]: Please insert a period at end of the paragraph.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**